TutorLizard.Web/Controllers/ScheduleItemController.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using TutorLizard.BusinessLogic.Data;
+using TutorLizard.BusinessLogic.Models;
+
+namespace TutorLizard.Web.Controllers
+{
+    public class ScheduleItemController : Controller
+    {
+        private readonly DataAccess _dataAccess = new();
+
+
+        // GET: ScheduleItemController
+        public ActionResult Index()
+        {
+            return View(_dataAccess.GetAllScheduleItems());
+        }
+
+        // GET: ScheduleItemController/Details/5
+        public ActionResult Details(int id)
+        {
+            var scheduleItem = _dataAccess.GetScheduleItemById(id);
+
+            if(scheduleItem == null)
+            {
+                return RedirectToAction("Error");
```

Comment:
We don't have `Error` action in this controller. I get 404. Maybe let's redirect to Index.